app/views/login.html

```diff
@@ -1 +1,17 @@
-<h1>Hi this is login page</h1>
+<form>
+  <div class="form-alert">
+    username not found
+  </div>
+  <div class="form-group">
+    <label for="username">Username</label>
+    <input id="username" type="text" required placeholder="ashketchum10">
```

Comment:
are you able to limit the chars to 20 like reddit does for login and register? and limit password to 72 as well?